From 3d80f9906703aeacfffbe53bffa23c8a444206f1 Mon Sep 17 00:00:00 2001
From: Duc Dang <dhdang@apm.com>
Date: Wed, 12 Aug 2015 17:01:12 -0700
Subject: [PATCH 01/27] MAINTAINERS: Add Applied Micro (APM) X-Gene Device Tree
 maintainer

This patch adds information of maintainers for Applied Micro (APM)
X-Gene device tree.

Signed-off-by: Duc Dang <dhdang@apm.com>
---
 MAINTAINERS | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 5f46784..f329509 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -788,6 +788,11 @@ S:	Maintained
 F:	drivers/net/appletalk/
 F:	net/appletalk/
 
+APPLIED MICRO (APM) X-GENE DEVICE TREE SUPPORT
+M:	Duc Dang <dhdang@apm.com>
+S:	Supported
+F:	arch/arm64/boot/dts/apm/
+
 APPLIED MICRO (APM) X-GENE SOC ETHERNET DRIVER
 M:	Iyappan Subramanian <isubramanian@apm.com>
 M:	Keyur Chudgar <kchudgar@apm.com>
-- 
2.6.2

